### PR TITLE
Cjl/feat/v18.5.1.0

### PR DESCRIPTION
**Pull request by lindsleycj**

### CODE REVIEW

File: cumulus/main.tf
Comment (mikedorfman) [inline cumulus/main.tf:1]: It looks like sqs_message_consumer_watcher_message_limit and sqs_message_consumer_watcher_time_limit are now configurable through the Cumulus module - do you mind making these configurable on the CIRRUS side as well?  I'm happy to open another PR to do this if that's better.

**Review by mikedorfman — COMMENTED** [review]
One comment, otherwise looks good.

File: cumulus/main.tf
Comment (lindsleycj) [reply cumulus/main.tf:1]: glad to do it here.  I'll get them added.

Comment (lindsleycj) [reply cumulus/main.tf:1]: @mikedorfman I figured out how to add the variables, but I'm guessing I need to pass them into the cumulus module in it's main.tf.  What variable am I supposed to pass?

```
xxx = var. sqs_message_consumer_watcher_message_limit
yyy = var. sqs_message_consumer_watcher_time_limit

Comment (lindsleycj) [reply cumulus/main.tf:1]: Ah, it looks like it was exposed after v18.5.1 was released.

Comment (lindsleycj) [reply cumulus/main.tf:1]: it's in the [v19.2.0-alpha.0](https://github.com/nasa/cumulus/releases/tag/v19.2.0-alpha.0) release

https://github.com/nasa/cumulus/commit/2fe02e67375b6d5b6553381dde0aac939751ec5f

Comment (mikedorfman) [reply cumulus/main.tf:1]: Ah, good call - thanks for digging into that!

**Review by reweeden — APPROVED** [review]

**Pull request merged**